app/templates/components/feed-item.hbs

```diff
@@ -1,15 +1,21 @@
 <div class="article-preview">
   <div class="article-meta">
-    <a href="profile.html"><img src= {{imageSrc}} alt="profile image for {{author}}"></a>
+    {{#link-to "profile" author}}
```

Comment:
i would link-to `author.id` over the actual author itself, it'll make the model hook run, it won't affect perf as the author object is most likely cached in the store.